lib/chai/utils/proxify.js

```diff
@@ -64,7 +64,7 @@ module.exports = function proxify (obj, nonChainableMethodName) {
         }
       }
 
-      return target[property];
+      return Reflect.get(target, property);
```

Comment:
FYI, `Reflect.get` is not different from regular computed access (unless you are using `receiver` parameter).
It also calls `[[Get]]` and `ToPropertyKey`. The only thing they are different is that `Reflect` methods throw on non-object `target` (however `target` here will always be an object). Same goes for `Reflect.has` above: `in` does the same thing. We can simplify checks and code by removing `Reflect` usages.